.github/workflows/ci.yml

```diff
@@ -90,7 +90,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        script: ['plume-lib', 'daikon']
+        script: ['plume-lib', 'daikon', 'misc']
```

Comment:
My suggestion [there](https://github.com/eisop/checker-framework/pull/1146#discussion_r1994480631) conflicts with `run: ./checker/bin-devel/test-${{ matrix.script }}.sh` below.

It is probably nicer to split off that task here too, to minimize when the extra dependencies are installed.